tests/positive/general/preemption.go

```diff
@@ -75,10 +75,15 @@ func makePreemptTest(components string) types.Test {
 
 	var systemFiles []types.File
 	for _, component := range []string{"b", "u"} {
+		var dir string
```

Comment:
This could be moved one scope further in, inside the `if enabled(component)` block.